content/docs/user-guide/managing-external-data.md

```diff
@@ -1,5 +1,10 @@
 # Managing External Data
 
+> ⚠️ Artifacts added with --external are not affected by
+> `dvc push/pull/status -c`. You are likely looking for
+> [straight-to-remote/cache](https://github.com/iterative/dvc/issues/4520)
+> functionality or `dvc import-url`
```

Comment:
Good note! If it's not urgent should we make this part of https://github.com/iterative/dvc.org/pull/2091#pullrequestreview-569642813 ?

And prob. link to /doc/command-reference/add#transferring-data-directly-to-the-remote instead.

Reply:
Cc @isidentical

Reply:
It will have to wait for 2.0 anyways, and even then people might try to use this. Better to have this note anyways.